h2o-py/h2o/model/regression.py

```diff
@@ -106,8 +106,8 @@ def h2o_r2_score(y_actual, y_predicted, weights=1.):
     :returns: R-squared (best is 1.0, lower is worse).
     """
     ModelBase._check_targets(y_actual, y_predicted)
-    numerator = (weights * (y_actual - y_predicted) ** 2).sum()
-    denominator = (weights * (y_actual - _colmean(y_actual)) ** 2).sum()
```

Comment:
this score method was ALWAYS returning `1.0` (that's how we cheat on benchmarks..) as `numerator` and `denominator` were still frames when the logic below expected numbers..

Reply:
most likely not used by anyone :)